pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -316,9 +313,11 @@ func (jc *MPIJobReconciler) onOwnerCreateFunc() func(event.CreateEvent) bool {
 			return true
 		}
 
+		logger := jc.Log.WithValues("namespace", mpiJob.Namespace, "name", mpiJob.Name)
```

Comment:
Is this namespace same as reconciler request namespace or different? If it's the same, can we insert the labelled logger which we set while initialising (in line 126) and use it from the context as is? If not, do we want to change the key name to something specific?